src/sarif-viewer.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeResults } from 'axe-core';
+import * as fs from 'fs';
+import { convertAxeToSarif } from './index';
+
+describe('Azure DevOps sarif viewer extension test', () => {
+    it('save sarif file to artifact to be shown by sarif viewer tab', () => {
+        const axeJSON: string = fs.readFileSync(
+            './src/test-resources/axe322-v2.dequemars-testsite.1554329251110.json',
+            'utf8',
+        );
+        const axeResultStub: AxeResults = JSON.parse(axeJSON) as AxeResults;
```

Comment:
Can you pick a better name for this variable? This isn't a stub; it's a real axe result from a test file.

Reply:
accept